planet/src/main/java/com/google/planet/servlets/UserServlet.java

```diff
@@ -0,0 +1,40 @@
+package com.google.planet.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/user")
+public class UserServlet extends HttpServlet {
+    private String userName;
+    private String userEmail;
+    private String userPhone;
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response)
+      throws IOException {
+        this.userName = request.getParameter("name");
```

Comment:
What's the scope of this memory? Does each user get their own instance of this class, or is it shared among users?

Reply:
This is also something I am working on in another PR. For now, the properties of this class are all static but I still have to further explore my options regarding the issue.

Reply:
so if I'm understanding correclty, this means that if two users made this request around the same time, User1 might get the profile page for User2, or vice versa?

We'll need to fix that before submitting. The best fix is probably to put the relevant information into the doGet request parameters and to remove the doPost functionality completely. Although an alternative would be to use a map indexed by userId to cache the info

Reply:
True! I am now working on that on another branch. Should I just remove the openProfile function from this PR and implement in a more detailed way in my current one?

Reply:
That's up to you. Either you can edit this PR to make it work for multiple users, or you can remove this feature entirely from this PR and add it in a future PR.

Reply:
Okay! I just deleted it and I will add it to another PR